content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -1,391 +1,287 @@
+
+```log
+LogRecord.Timestamp:               2023-08-14T06:44:53.9279186Z
+LogRecord.TraceId:                 3961d22b5f90bf7662ad4933318743fe
+LogRecord.SpanId:                  93d5fcea422ff0ac
+LogRecord.TraceFlags:              Recorded
+LogRecord.CategoryName:            simple-dotnet
+LogRecord.LogLevel:                Information
+LogRecord.StateValues (Key:Value):
+    result: 1
+    OriginalFormat (a.k.a Body): Anonymous player is rolling the dice: {result}
+
+Resource associated with LogRecord:
+service.name: simple-dotnet
+telemetry.auto.version: 0.7.0
+telemetry.sdk.name: opentelemetry
+telemetry.sdk.language: dotnet
+telemetry.sdk.version: 1.4.0.802
+
+LogRecord.Timestamp:               2023-08-14T06:44:53.9279186Z
+LogRecord.TraceId:                 3961d22b5f90bf7662ad4933318743fe
+LogRecord.SpanId:                  93d5fcea422ff0ac
+LogRecord.TraceFlags:              Recorded
+LogRecord.CategoryName:            simple-dotnet
+LogRecord.LogLevel:                Information
+LogRecord.StateValues (Key:Value):
+    result: 1
+    OriginalFormat (a.k.a Body): Anonymous player is rolling the dice: {result}
+
+Resource associated with LogRecord:
+service.name: simple-dotnet
+telemetry.auto.version: 0.7.0
+telemetry.sdk.name: opentelemetry
+telemetry.sdk.language: dotnet
+telemetry.sdk.version: 1.4.0.802
+
+info: simple-dotnet[0]
+      Anonymous player is rolling the dice: 1
+Activity.TraceId:            3961d22b5f90bf7662ad4933318743fe
+Activity.SpanId:             93d5fcea422ff0ac
 Activity.TraceFlags:         Recorded
-Activity.ParentSpanId:       acbff23f5ad721ff
-Activity.ActivitySourceName: MyService
-Activity.DisplayName:        SayHello
-Activity.Kind:               Internal
-Activity.StartTime:          2023-02-21T12:27:41.9596458Z
-Activity.Duration:           00:00:00.0005683
+Activity.ActivitySourceName: OpenTelemetry.Instrumentation.AspNetCore
+Activity.DisplayName:        /rolldice
+Activity.Kind:               Server
+Activity.StartTime:          2023-08-14T06:44:53.9278162Z
+Activity.Duration:           00:00:00.0049754
 Activity.Tags:
-    foo: 1
-    bar: Hello, World!
-    baz: [1,2,3]
+    net.host.name: localhost
+    net.host.port: 8080
+    http.method: GET
+    http.scheme: http
+    http.target: /rolldice
+    http.url: http://localhost:8080/rolldice
+    http.flavor: 1.1
+    http.user_agent: curl/8.0.1
+    http.status_code: 200
 Resource associated with Activity:
-    service.name: MyService
-    service.instance.id: 2b07a9ca-29c4-4e01-b0ed-929184b32192
+    service.name: simple-dotnet
+    telemetry.auto.version: 0.7.0
+    telemetry.sdk.name: opentelemetry
+    telemetry.sdk.language: dotnet
+    telemetry.sdk.version: 1.4.0.802
 ```
 
-</details>
-
-You'll notice the `Activity` objects from ASP.NET Core alongside the `Activity`
-we created manually in our controller action.
+At step 6, when stopping the server, you should see an output of all the metrics
```

Comment:
nit: the metric console exporter should be exporter every few seconds, so maybe put a note saying app should be run for 20-30 seconds, so as to see metrics as well..(that would be better for all signals, as OTLP exporter for trace/log also does the batching)

Reply:
Part of c8693f4f3d7da9affbc4d69fc5bc9abedaf92f21